src/pages/search/index.js

```diff
@@ -263,32 +224,25 @@ export const Search = ({
   )
 }
 
-Search.getInitialProps = async ({ query }) => {
-  const {
-    search: {
-      commonQueries: {
-        ordering,
-        num_downloadable_samples__gt: numDownloadableSamples
-      }
-    }
-  } = options
+export const getServerSideProps = async ({ query }) => {
+  const queryParams = getSearchQueryForAPI(query)
   const filterOrders = query.filter_order ? query.filter_order.split(',') : []
-  const queryParams = {
-    ...getSearchQueryForAPI(query),
-    limit: query.limit || 10,
-    offset: query.offset * query.limit || 0,
-    ordering: query.sortby || ordering,
-    ...(query.search ? { search: query.search } : {}),
-    num_downloadable_samples__gt: !query.empty
-      ? Number(numDownloadableSamples.hide)
-      : Number(numDownloadableSamples.show)
-  }
+
   const response = await fetchSearch(queryParams, filterOrders)
 
-  return {
-    query: queryParams,
-    ...response
+  if (response.ok && response) {
+    return {
+      props: {
+        query: formatFacetQueryParams(
```

Comment:
Can we add a comment here because we modify the api response. I think something like the following will do: 
```js
// There is a discrepancy between query param names for facet filters. So we adjust them to the facet names to pass to the page.
```

And then where we convert them back just describe the reverse process.